target/linux/ath79/dts/qca9563_comfast_cf-e375ac.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "COMFAST CF-E375AC";
+	compatible = "comfast,cf-e375ac", "qca,qca9563";
+
+	aliases {
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-running = &led_lan;
+		led-upgrade = &led_lan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan: led_lan {
```

Comment:
The idea is actually that the DT _label_, i.e. the thing before the colon, has led_ prefix as it's used in global context. The node names don't need that, as they only exist inside the leds node. So
led_lan: lan {
or just
wlan5g {
...